README.md

```diff
@@ -64,7 +64,7 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
        // Override point for customization after application launch.
         let mParticleOptions = MParticleOptions(key: "<<<App Key Here>>>", secret: "<<<App Secret Here>>>")
         
-       //Please see the Identity page for more information on building this object
+       /* Optional - Please see the Identity page for more information on building this object
```

Comment:
same as below, instead of commenting the whole section of code out, can you just add a comment